api/src/main/java/org/pmiops/workbench/config/Environment.java

```diff
@@ -0,0 +1,29 @@
+package org.pmiops.workbench.config;
+
+import com.google.appengine.api.utils.SystemProperty;
+import com.google.appengine.api.utils.SystemProperty.Environment.Value;
+
+public class Environment {
```

Comment:
Where are the environment variables set up? Is that going to be in the config or automatically generated somehow?

Reply:
Right now everything in this when running in AppEngine comes from SystemProperty (see default constructor.)

The main purpose of this is to allow unit tests to set different values for Environment using the other constructor, and exercise the other cases.